cmd/registry-replacer/main.go

```diff
@@ -278,6 +304,15 @@ func replacer(
 			if err := pruneUnusedReplacements(config, allReplacementCandidates); err != nil {
 				return fmt.Errorf("failed to prune unused replacements: %w", err)
 			}
+
+			// to prune base images we'll need the fully-resolved config.
+			resolvedConfig, err := configResolver()
+
+			if err != nil {
```

Comment:
I think it would be good to return the error, otherwise we won't notice if this starts failing

Reply:
I guess I was thinking that we wouldn't want to abort and stop up the rest of the process just because we couldn't resolve the config and therefore not prune the base images, though maybe I was just being overly paranoid. I can definitely return the error here

Reply:
I mean in the beginning this might be ok but once everything worked we need to remove it because otherwise we will not notice errors here, as no one looks at successful jobs. Maybe initially merge it like this and do a follow-up when the job is green?

Reply:
That sounds good to me. I definitely agree that it _should_ return the actual error. I'll merge it like this and follow up with that like you said.